src/codemirrorCommands.ts

```diff
@@ -97,198 +103,235 @@ export class VimCellManager {
         }: IKeybinding) => {
           if (keybindEnabled) {
             if (mapfn === 'map') {
-              lvim.map(command, keys, context);
+              Vim.map(command, keys, context);
             } else {
-              lvim.noremap(command, keys, context);
+              Vim.noremap(command, keys, context);
             }
           }
         }
       );
 
       Vim.handleKey(lcm, '<Esc>');
```

Comment:
I don't see why this doesn't put the fileeditor into normal mode. It really seems as though it should be

Reply:
There is a race condition, when it first fires the file editor has not initialised yet.

Reply:
[Should ](https://github.com/jupyterlab-contrib/jupyterlab-vim/pull/90/commits/bf589e5dfc919446acfa99c55bb453e38526cc9e)